### PR TITLE
Add description for Kafka multi-topic feature

### PR DESCRIPTION
**Pull request by zhaoxinyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -569,22 +569,68 @@ worker-num = 16
...
+占位符 `{schema}` 用来匹配库名, 占位符 `{table}` 用来匹配表名，二者均为小写，即不允许出现 `{ScHeMa}` 以及 `{TaBLe}` 这样的占位符。
+
+一些示例如下：
+
+- `matcher = ['test1.table1', 'test2.table2'], topic = "hello_{schema}_{table}"`
+    - 对于表 `test1.table1` 对应的数据变更事件，发送到名为 `hello_test1_table1` 的 topic 中
+    - 对于表 `test2.table2` 对应的数据变更时间，发送到名为 `hello_test2_table2` 的 topic 中
+- `matcher = ['test3.*', 'test4.*'], topic = "{schema}_hello"`
+    - 对于 `test3` 下的所有表对应的数据变更事件，发送到名为 `test3_hello` 的 topic 中
+    - 对于 `test4` 下的所有表对应的数据变更事件，发送到名为 `test4_hello` 的 topic 中
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:615]:
```suggestion
- `matcher = ['test3.*', 'test4.*'], topic = "hello_{schema}_world"`
    - 对于 `test3` 下的所有表对应的数据变更事件，发送到名为 `hello_test3_world` 的 topic 中
    - 对于 `test4` 下的所有表对应的数据变更事件，发送到名为 `hello_test4_ world` 的 topic 中
- `matcher = ['*.*'], topic = "{schema}_{table}"`
    - 对于 TiCDC 监听的所有表，按照“库名_表名”的规则分别分发到独立的 topic 中；例如对于 `test.account` 表，TiCDC 会将其数据变更日志分发到名为 `test_account` 的 Topic 中。
```

**Review by leoppro — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -569,22 +569,68 @@ worker-num = 16
...
-# - index-value：以表的主键或者唯一索引的值做 Hash 计算并进行 event 分发
-# - table：以表的 schema 名和 table 名做 Hash 计算并进行 event 分发
-# matcher 的匹配语法和过滤器规则语法相同
+# 支持 partition 及 topic 两种 event 分发器。二者的详细说明见以下两小节。
+
+# matcher 的匹配语法和过滤器规则语法相同，matcher 规则匹配的几种情况如下所示：
+# - 对于匹配了 matcher 规则的表，按照对应的 topic 表达式指定的策略进行分发。例如表 test3.aa，按照 topic 表达式 2 分发；表 test5.aa，按照 topic 表达式 3 分发。
+# - 对于匹配了多个 matcher 规则的表，以靠前的 matcher 对应的 topic 表达式为准。例如表 test1.aa，按照 topic 表达式 1 分发。
+# - 对于没有匹配任何 matcher 的表，将对应的数据变更事件发送到 --sink-uri 中指定的默认 topic 中。例如表 test10.aa 发送到默认 topic。
+# - 对于匹配了 matcher 规则但是没有指定 topic 分发器的表，将对应的数据变更发送到 --sink-uri 中指定的默认 topic 中。例如表 test6.aa 发送到默认 topic。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:578]: 当 【自定义 Kafka Sink 的 Topic 和 Partition 的分发规则】这一章独立出来后，这里的注释就可以适当简化。

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -569,22 +569,68 @@ worker-num = 16
...
+
 # 对于 MQ 类的 Sink，可以指定消息的协议格式
 # 目前支持 canal-json、open-protocol、canal、avro 和 maxwell 五种协议。
 protocol = "canal-json"
-
 ```
 
+### Partition 分发器
+
+partition 分发器用 dispatcher = "xxx" 来指定，支持 default、ts、rowid、table 四种 partition 分发器，分发规则如下：
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:593]: 这里的 rowid，但是下面列表中是 index-value。是不是要改成 index-value？

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -569,22 +569,68 @@ worker-num = 16
...
+
+- default：有多个唯一索引（包括主键）时按照 table 模式分发；只有一个唯一索引（或主键）按照 rowid 模式分发；如果开启了 old value 特性，按照 table 分发
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:595]: 这句话中的 rowid 也要改成 index-value 吗？

File: ticdc/manage-ticdc.md (hunk 3 above)
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:593]: 看代码这里的 rowid 跟 index-value 应该是互为别名，还在确认中

Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md]: @leoppro ，看了一下历史记录，5.0上 rowid 跟 index-value 开始就互为别名，我们后边以那个为准？

Comment (leoppro) [reply ticdc/manage-ticdc.md]: 麻烦在文档统一为 index-value

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -569,27 +569,104 @@ worker-num = 16
...
-# - ts：以行变更的 commitTs 做 Hash 计算并进行 event 分发
-# - index-value：以表的主键或者唯一索引的值做 Hash 计算并进行 event 分发
-# - table：以表的 schema 名和 table 名做 Hash 计算并进行 event 分发
-# matcher 的匹配语法和过滤器规则语法相同
+# 支持 partition 及 topic 两种 event 分发器。二者的详细说明见下一节。
+# matcher 的匹配语法和过滤器规则语法相同，matcher 匹配规则的详细说明见下一节。
 dispatchers = [
-    {matcher = ['test1.*', 'test2.*'], dispatcher = "ts"},
-    {matcher = ['test3.*', 'test4.*'], dispatcher = "rowid"},
+    {matcher = ['test1.*', 'test2.*'], topic = "Topic 表达式 1", dispatcher = "ts" },
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:575]:
```suggestion
    {matcher = ['test1.*', 'test2.*'], topic = "Topic 表达式 1", partition = "ts" },
```
麻烦再文档统一为 partition，不再提及 dispatcher

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -569,27 +569,104 @@ worker-num = 16
...
+### Matcher 匹配规则
+
+以上一节示例配置文件中的 dispatchers 配置项为例：
+
+- 对于匹配了 matcher 规则的表，按照对应的 topic 表达式指定的策略进行分发。例如表 test3.aa，按照 topic 表达式 2 分发；表 test5.aa，按照 topic 表达式 3 分发。
+- 对于匹配了多个 matcher 规则的表，以靠前的 matcher 对应的 topic 表达式为准。例如表 test1.aa，按照 topic 表达式 1 分发。
+- 对于没有匹配任何 matcher 的表，将对应的数据变更事件发送到 --sink-uri 中指定的默认 topic 中。例如表 test10.aa 发送到默认 topic。
+- 对于匹配了 matcher 规则但是没有指定 topic 分发器的表，将对应的数据变更发送到 --sink-uri 中指定的默认 topic 中。例如表 test6.aa 发送到默认 topic。
+
+### Topic 分发器
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:602]: 麻烦补充 partition 分发器

Comment (leoppro) [reply ticdc/manage-ticdc.md:602]: 哦 我在下面看到了 抱歉

File: ticdc/manage-ticdc.md (hunk 4 above)
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md]: Done

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**